packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -203,125 +203,130 @@ const makeRenderer = (
 	scripts: Array<string> = [],
 	enableServiceWorker: boolean,
 	cssLinks: ?Array<string>
-) => (request: HapiRequest, reply: HapiReply): Promise<RenderResult> => {
-	middleware = middleware || [];
+) => {
+	const routesPromise = resolveAllRoutes(routes);
+	return (request: HapiRequest, reply: HapiReply): Promise<RenderResult> => {
```

Comment:
new setup created an indentation change https://github.com/meetup/meetup-web-platform/pull/477/files?w=1 to see the diff without whitespace changes